docs/content/Workspace/Auto-Suspend.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Auto-suspend
+permalink: /workspace/auto-suspend
+category: Workspace
```

Comment:
Do you think this really belongs to Workspace? Sounds like the Deployment stuff to me :-)

Reply:
I wasn't sure myself TBH, was actually hoping you'd have a suggestion 😀 Will move it to the Deployment category, I agree, its a better fit